hamilton_launch_board/src/views/pages/VideoFeed.vue

```diff
@@ -1,17 +1,40 @@
 <template>
-  <v-jumbotron>
-    <v-container fill-height>
-      <h1 class="display-3">{{ msg }}</h1>
+    <v-container fluid grid-list-md>
+        <h1 class="display-3">{{ title }}</h1>
+        <v-layout row wrap>
+            <v-flex lg6>
+                <videostream stream-file="test1.mjpg">
+                </videostream>
+            </v-flex>
+            <v-flex lg6>
+                <v-layout row wrap>
+                    <v-flex md7>
+                        <videostream
+                          stream-file="test2.mjpg">
+                        </videostream>
+                    </v-flex>
+                    <v-flex md7>
+                        <videostream
+                          stream-file="test3.mjpg">
```

Comment:
Let's not name these tests. Naming them what they actually are would be best if possible. Like `mainView.mjpg`, `fillValveView.mjpg`